libr/main/rabin2.c

```diff
@@ -57,6 +57,7 @@ static int rabin_show_help(int v) {
 		" -s              symbols\n"
 		" -S              sections\n"
 		" -SS             segments\n"
+		" -SSS            sections mapping to segments\n"
```

Comment:
add `-SSS` to line 32 as well, as `-g` option is a way to say "all flags" and -SSS should be listed there as well since we have added R_BIN_REQ_SECTIONS_MAPPING to the 'g' flag.